core/src/autogluon/core/__init__.py

```diff
@@ -16,3 +16,8 @@
 __logging.getLogger("distributed.utils_perf").setLevel(__logging.ERROR)
 __logging.getLogger("distributed.logging.distributed").setLevel(__logging.ERROR)
 __logging.getLogger("distributed.worker").setLevel(__logging.ERROR)
+
+try:
+    from .version import __version__
+except ImportError:
+    pass
```

Comment:
I'm not sure if we need to pass here.

Reply:
Are you suggesting we raise an exception if the version file does not exist? If not, could you mention the code you are suggesting it be changed to?

Reply:
I mean, we can directly change to 

```
from .version import __version__
```

Reply:
updated